scout/server/blueprints/genes/controllers.py

```diff
@@ -26,7 +26,8 @@ def genes(store, query):
     nr_genes_37 = store.nr_genes(build="37")
     nr_genes_38 = store.nr_genes(build="38")
     genes_subset = list(store.all_genes(add_transcripts=False, limit=20))
-    last_updated = document_generated(genes_subset[0]["_id"] if genes_subset else None)
+    last_inserted_gene = store.hgnc_collection.find({}).sort([("timestamp", -1)]).limit(1)
```

Comment:
:+1: that should work!

Reply:
(I never realised the ObjectIds themselves were part creation time stamp! Nothing preventing the rest of the document from changing I suppose, but I think we drop the relevant documents when updating, right?)

Reply:
> Tested locally:
> 
> <img alt="image" width="530" src="https://user-images.githubusercontent.com/28093618/167103702-eb225f1c-58f2-44c9-b825-690cd917d533.png">

Very few hg38 genes you have there! 😄

Reply:
I don't think my previous version worked? 

It selected a document created at 15 something.

If you instead use `.sort('_id',-1).limit(1)` it returns a document created after (16 something)